src/endpoint/s3/s3_rest.js

```diff
@@ -264,7 +264,8 @@ function _get_method_from_req(req) {
     const s3_op = s3_bucket_policy_utils.OP_NAME_TO_ACTION[req.op_name];
     if (!s3_op) {
         dbg.error(`Got a not supported S3 op ${req.op_name} - doesn't suppose to happen`);
-        throw new S3Error(S3Error.InternalError);
+        // Added access denied error to align with Ceph test test_post_object_wrong_bucket
+        throw new S3Error(S3Error.AccessDenied);
```

Comment:
There are two things
1. test_post_object_wrong_bucket ceph test should be black listed because `post_bucket` action is not supported. I will add this test to black list.
2. This test was working before because request was considered anonymous and anonymous request was returning 403 error before.
Ceph is passing `AWSAccessKeyId` and `signature` as payload and But `make_auth_token_from_request()` have a [check](https://github.com/noobaa/noobaa-core/blob/master/src/util/signature_utils.js#L269) `if (req.query.AWSAccessKeyId && req.query.Signature)` where `req.query` is empty.